libs/api/domains/municipalities-financial-aid/src/lib/municipalitiesFinancialAid.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common'
+import { MunicipalitiesFinancialAidResolver } from './municipalitiesFinancialAid.resolver'
+import { MunicipalitiesFinancialAidService } from './municipalitiesFinancialAid.service'
+
+import {
+  ApplicationApiProvider,
+  MunicipalityApiProvider,
+  FilesApiProvider,
+} from '@island.is/clients/municipalities-financial-aid'
+
+@Module({
+  providers: [
+    MunicipalitiesFinancialAidResolver,
+    MunicipalitiesFinancialAidService,
+    ApplicationApiProvider,
+    MunicipalityApiProvider,
+    FilesApiProvider,
```

Comment:
Should not provide these here, but instead have something like `imports: [MunicipalitiesFinancialAidClientModule]` and export the apis on the other side.